iot-hub/Quickstarts/InvokeDeviceMethod/Parameters.cs

```diff
@@ -23,7 +24,7 @@ internal class Parameters
             "DeviceId",
             Required = false,
             HelpText = "The Id of the device to receive the direct method." +
-            "\nDefaults to \"MyDotnetDevice\".")]
+            "\nDefaults to 'MyDotnetDevice'.")]
```

Comment:
We should instead consider using the `Default` attribute property, instead of initializing it in 28.